lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -5,15 +5,12 @@ import { GithubTagsDatasource } from '../../datasource/github-tags';
 import { HexpmBobDatasource } from '../../datasource/hexpm-bob';
 import { NodeDatasource } from '../../datasource/node';
 import { RubyVersionDatasource } from '../../datasource/ruby-version';
-import * as nodeVersioning from '../../versioning/node';
 import * as regexVersioning from '../../versioning/regex';
 import * as semverVersioning from '../../versioning/semver';
 import type { PackageDependency } from '../types';
 
 export type StaticTooling = Partial<PackageDependency> &
-  Required<
-    Pick<PackageDependency, 'datasource' | 'versioning' | 'packageName'>
-  >;
+  Required<Pick<PackageDependency, 'datasource'>>;
```

Comment:
According to our current approach, shouldn't `depName` be mandatory, and `packageName` optional? I'm seeing instances below where only `packageName` is specified

Reply:
depName is extracted from source, so node is the only dep where it's overwritten from nodejs to node.